### PR TITLE
[Docs] Spark engine support for steps: MapValues, Imputer, OneHotEncoder, DropFeatures; DateExtractor (partial) 

### PR DESCRIPTION
**Pull request by jillnogold**
ML-2709. For discussion, see https://github.com/mlrun/mlrun/pull/2991

### CODE REVIEW

File: docs/serving/available-steps.md [hunk 1]
```diff
@@ -38,11 +38,12 @@ Click on the step names in the following sections to see the full usage.
...
+| [mlrun.feature_store.DropFeatures](../api/mlrun.feature_store.html#mlrun.feature_store.steps.DropFeatures) | Drop features from feature list. Supported also by the Pandas and Spark engines. |
+| [mlrun.feature_store.Imputer](../api/mlrun.feature_store.html#mlrun.feature_store.steps.Imputer) | Replace None values with default values. Supported also by the Pandas and Spark engines. |
+| [mlrun.feature_store.MapValues](../api/mlrun.feature_store.html#mlrun.feature_store.steps.MapValues) | Map column values to new values. Supported also by the Pandas and Spark engines. |
+| [mlrun.feature_store.OneHotEncoder](../api/mlrun.feature_store.html#mlrun.feature_store.steps.OneHotEncoder) | Create new binary fields, one per category (one hot encoded). Supported also by the Pandas and Spark engines. | 
+| [mlrun.feature_store.SetEventMetadata](../api/mlrun.feature_store.html#mlrun.feature_store.steps.SetEventMetadata) | Set the event metadata (id, key, timestamp) from the event body. Supported also by the Pandas and Spark engines.|
```
Comment (davesh0812) [inline docs/serving/available-steps.md:46]: SetEventMetadata supported only by storey

File: docs/serving/available-steps.md [hunk 2]
```diff
@@ -101,5 +102,5 @@ Click on the step names in the following sections to see the full usage.
 ## Other
 | Class name                                       | Description                                   |   
 |--------------------------------------------------|-----------------------------------------------------------|
-| [mlrun.feature_store.FeaturesetValidator](../api/mlrun.feature_store.html#mlrun.feature_store.steps.FeaturesetValidator) | Validate feature values according to the feature set validation policy. | 
+| [mlrun.feature_store.FeaturesetValidator](../api/mlrun.feature_store.html#mlrun.feature_store.steps.FeaturesetValidator) | Validate feature values according to the feature set validation policy. Supported also by the Pandas and Spark engines. |
```
Comment (davesh0812) [inline docs/serving/available-steps.md:105]:
```suggestion
| [mlrun.feature_store.FeaturesetValidator](../api/mlrun.feature_store.html#mlrun.feature_store.steps.FeaturesetValidator) | Validate feature values according to the feature set validation policy. Supported also by the Pandas engines. | 
```

**Review by davesh0812 — COMMENTED** [review]
:)

File: docs/serving/available-steps.md (hunk 1 above)
Comment (jillnogold) [reply docs/serving/available-steps.md]: deleted

**Review by davesh0812 — APPROVED** [review]
LGTM :)

**Pull request merged**